knowage-vue/src/modules/documentExecution/dashboard/widget/WidgetEditor/WidgetEditorSettingsTab/common/styleToolbar/WidgetEditorStyleIcon.vue

```diff
@@ -41,18 +42,18 @@ export default defineComponent({
             active: false,
             iconPickerDialogVisible: false,
             displayValue: '',
-            color: null as { r: number; g: number; b: number } | null,
+            color: null as { r: number; g: number; b: number; a: number } | null,
             newColor: 'rgb(255, 255, 255)',
             colorPickTimer: null as any,
             useClickOutside
         }
     },
     computed: {
         showArowDown() {
```

Comment:
should this be showArrowDown?

Reply:
I will look into it.

Reply:
@Redjaw what did you mean by the first comment? What should be arrow-down?

Reply:
If it is just the rename, I just did it. (Renamed to showArrowDown)